src/main/java/com/klodnicki/Bike/v2/exception/CustomExceptionHandler.java

```diff
@@ -16,7 +15,9 @@ public class CustomExceptionHandler {
     public ResponseEntity<String> handleMethodArgumentNotValidException(MethodArgumentNotValidException ex) {
         Object[] detailMessageArguments = ex.getDetailMessageArguments();
 
-        assert detailMessageArguments != null;
-        return new ResponseEntity<>("Something went wrong!: \n" + detailMessageArguments[1], HttpStatus.BAD_REQUEST);
+        if (detailMessageArguments != null) {
+            return new ResponseEntity<>("Something went wrong!: \n" + detailMessageArguments[1], HttpStatus.BAD_REQUEST);
+        }
+        return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
I think that regardless of anything, when you catch exception REST API shouldn't return status 200.

Reply:
of course you are right! I guess I haven't woken up yet;)